lib/elastic_record/version.rb

```diff
@@ -0,0 +1,11 @@
+module ElasticRecord
+  class Version
+    include Model
+
+    def self.es6?
+      @es6_mode ||= elastic_connection.json_get('/')
+        .dig('version', 'number')
+        .start_with?('6.')
+    end
+  end
+end
```

Comment:
Creating a new class to include Model seems a little heavyweight just so you can access one method (although I appreciate the API). Maybe add a class method on Model instead?

Reply:
Well, the idea behind the class was to make it accessible outside of the scope of a specific model/index.

Reply:
I was thinking `ElasticRecord::Model.es6?` but your change to the `ElasticConnection` concern is fine.